modules/katamari/src/test/ts/atomic/api/obj/ObjIsEmptyTest.ts

```diff
@@ -20,4 +22,12 @@ describe('atomic.katamari.api.obj.ObjIsEmptyTest', () => {
       }
     ));
   });
+
+  it('helps TypeScript understand the empty type', () => {
```

Comment:
Could we also add some form of tests that checks when it's empty. Like `if (isEmpty({})) { assert.deepEqual({}, {}) }` or something os that it checks that result type became an empty object.

Reply:
TypeScript allows assigning any values to a `{}`	type, so it's mostly a pointless test, but sure.